### DISCUSSION TITLE
using external firebird databases as cluster.

### DISCUSSION THREAD

**Discussion by gladiston** [FirebirdSQL/firebird#8022] [Ideas]
Is it possible in the future all firebird databases will an unique cluster of databases and do this:
select a.name, b.pic_logo from database1..clients a
inner join database2..images b on (a.id_client=b.id_client)

Maybe using databases aliases in databases.conf as alias database name in query.
Separate databases is convenient to be written in different devices where the performance will be better for each database and backups too.

I like this feature in another database like MSSQL.

Just an idea.

**Comment by romansimakov** [comment]
Sure it's possible. At least in Red Soft we are already working on this and RedDatabase 6 must get this future. Then it will be ported to Firebird as well.
